doc/src/modules/numeric-computation.rst

```diff
@@ -1,3 +1,5 @@
+.. _numeric_computation:
```

Comment:
This page doesn't seem to mention autowrap or all of the functions in the autowrap module.